server/controllers/article.js

```diff
@@ -73,16 +74,24 @@ class ArticleController {
    * @description returns all article.
   */
   static getAll(req, res) {
```

Comment:
you need to write a test for this. I think this function returns the same two article each time

Reply:
Alright, thanks